### PR TITLE
fix(Core\Scripts):Veridian Broodlings SAI moved into C++ and fixed movement issue after casting

### PR DESCRIPTION
**Pull request by Ragelezz**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8603
- Closes https://github.com/chromiecraft/chromiecraft/issues/2128

## Tests Performed:

- .go c id 17589 and let him cast Poison Bolt, notice he won't come and stay there
- 

## How to Test the Changes:

1. .go c id 17589 and let him cast Poison Bolt, notice he now comes to melee swing you when he doesn't cast
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Nyeriah** [comment]
Hey Rageless,

I appreciate the time you've put into this, but may I suggest another approach to this issue?

Those mobs seem to have the same issue I addressed here: https://github.com/azerothcore/azerothcore-wotlk/pull/8584

They have SMARTCAST_COMBAT_MOVEMENT flag set to their spell (flag 64), that means they won't move in melee range ever unless they're out of mana.

```sql
UPDATE `smart_scripts` SET `action_param2` = 0, `comment` = 'Veridian Broodling - In combat - Spellcast Poison Bolt' WHERE `entryorguid`=17589 AND `source_type`=0 AND `id` = 0;
```

Could you check if the query above solves the issue you're trying to address?

Thanks!

File: data/sql/updates/pending_db_world/rev_1634719155364267500.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1634719155364267500');
+
+
+
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1634719155364267500.sql:4]:
```suggestion

```

File: data/sql/updates/pending_db_world/rev_1634719155364267500.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1634719155364267500');
+DELETE FROM `smart_scripts` WHERE `entryorguid`=17589 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES (17589, 0, 0, 0, 0, 0, 100, 0, 0, 0, 12000, 14000, 0, 11, 21067, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Veridian Broodling - In combat - Spellcast Poison Bolt');
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1634719155364267500.sql:3]:
```suggestion
INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1634719155364267500');

DELETE FROM `smart_scripts` WHERE `entryorguid`=17589 AND `source_type`=0;
INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
(17589, 0, 0, 0, 0, 0, 100, 0, 0, 0, 12000, 14000, 0, 11, 21067, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Veridian Broodling - In combat - Spellcast Poison Bolt');
```

**Review by Nyeriah — APPROVED** [review]

**Comment by Nyeriah** [comment]
Hello,

Moving this one to the to-be-merged list as I believe it has been tested enough. Thanks for your PR!

**Comment by T1ti** [comment]
I strongly recommand using https://github.com/BAndysc/WoWDatabaseEditor over keira3 to edit SAI, it is much better

**Comment by FrancescoBorzi** [comment]
@T1ti offtopic: I'd like to know what's missing in the Keira3's SAI Editor so we can improve it

**Pull request merged**